src/vos/README.md

```diff
@@ -1,13 +1,13 @@
 
 # Versioning Object Store
 
-The Versioning Object Store (VOS) is responsible for providing and maintaining a persistent object store that supports byte-granular access and versioning.
+The Versioning Object Store (VOS) is responsible for providing and maintaining a persistent object store that supports byte-granular access and versioning for a single shard in a DAOS pool.
```

Comment:
Is there other documentation explaining what a shard and pool are? Maybe a reference to those?